dashboard/src/modules/components/ProfileComponent/NewKeyModal.jsx

```diff
@@ -21,7 +14,12 @@ const NewKeyModal = (props) => {
   const { isModalOpen, newKeyLabel } = useSelector(
     (state) => state.keyManagement
   );
-
+  const handleKeyPress = (e) => {
+    const key = e.key;
+    if (key === "Enter" && newKeyLabel) {
+      dispatch(sendNewKeyRequest(newKeyLabel));
+    }
```

Comment:
So, if I have the modal open and I don't enter a label, then `newKeyLabel` is `""` which evaluates to `false` and it doesn't call `dispatch()` here, right?  Is that desirable?

We permit keys with no label, so...we're going to force users to click in order to create them?  (Or, what is the check of `newKeyLabel` here supposed to accomplish?)

Reply:
Oops! I completely forgot `newKeyLabel ` can be empty